lib/app/features/posts/view/favorites_view.dart

```diff
@@ -27,6 +28,7 @@ class FavoritesView extends StatelessWidget {
                             onTap: () => context.pushNamed("details", extra: post),
```

Comment:
yeah, apologize for the delay on this one, this should be somethign like context.pushNamed(DetailsPage.routeName)